bubblewrap.c

```diff
@@ -75,6 +75,30 @@ int opt_info_fd = -1;
 int opt_seccomp_fd = -1;
 char *opt_sandbox_hostname = NULL;
 
+#define CAP_TO_MASK_0(x) (1L << ((x) & 31))
+#define CAP_TO_MASK_1(x) CAP_TO_MASK_0(x - 32)
+
+#define ALLOWED_CAPS0_UNSHARE_USER (CAP_TO_MASK_0 (CAP_CHOWN) | CAP_TO_MASK_0(CAP_SETUID) | CAP_TO_MASK_0 (CAP_SETGID) | \
+                                    CAP_TO_MASK_0 (CAP_FOWNER) | CAP_TO_MASK_0 (CAP_DAC_OVERRIDE) | CAP_TO_MASK_0 (CAP_FSETID) | \
+                                    CAP_TO_MASK_0 (CAP_SYS_ADMIN) | CAP_TO_MASK_0 (CAP_SETFCAP))
+#define ALLOWED_CAPS1_UNSHARE_USER (0)
```

Comment:
Its not entirely clear to me what these allowed lists are for. First of all, if we're unprivileged, then that means we're always using --unshare-user. So, do we need a special check for that? Additionally, i thought the kernel is meant to guarantee that all possible caps are safe in a user namespace, no?

I guess this is for the case where you run bubblewrap as root?